internal/commands/root/auth.go

```diff
@@ -0,0 +1,125 @@
+// Copyright © 2020 The virtual-kubelet authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package root
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apiserver/pkg/authentication/authenticator"
+	"k8s.io/apiserver/pkg/authentication/authenticatorfactory"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+	"k8s.io/apiserver/pkg/authorization/authorizerfactory"
+	"k8s.io/apiserver/pkg/server/dynamiccertificates"
+	clientset "k8s.io/client-go/kubernetes"
+	authenticationclient "k8s.io/client-go/kubernetes/typed/authentication/v1"
+	authorizationclient "k8s.io/client-go/kubernetes/typed/authorization/v1"
+
+	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/config"
+	kubeletserver "k8s.io/kubernetes/pkg/kubelet/server"
```

Comment:
This is probably fine for now... but all of what we are using from this package is trivial. I'd prefer to not import from kubelet if we don't have to.

Reply:
thanks for your suggestion, this dependency is removed.